bjoern/request.c

```diff
@@ -180,7 +180,22 @@ on_header_value(http_parser* parser, const char* value, size_t len)
   if(!PARSER->invalid_header) {
     /* Set header, or append data to header if this is not the first call */
     _set_or_append_header(REQUEST->headers, PARSER->field, value, len);
+
+    if (parser->http_major > 0 && parser->http_minor > 0) {
```

Comment:
Minor optimisation: We can skip the string comparison if any of `.expect*` is already true